lib/modern_searchlogic/scope_procedure.rb

```diff
@@ -2,7 +2,16 @@ module ModernSearchlogic
   module ScopeProcedure
     def self.included(base)
       base.singleton_class.class_eval do
-        alias_method :scope_procedure, :scope
+        def scope_procedure(name, options = nil)
+          define_singleton_method name do |*args|
+            case options
+            when Symbol
+              send(options)
```

Comment:
minor point but should this be `public_send`? and should it pass in `*args`? (maybe I don't understand the use case for the `scope_procedure :foo, :bar` formulation)

Reply:
> minor point but should this be public_send?

Ah, yeah, probably.

> and should it pass in *args? 

Another probably here as well.

> (maybe I don't understand the use case for the scope_procedure :foo, :bar formulation)

I don't either really see the point either. We can take it out and see what happens upstream. 

I'll get the other changes in place. Let me know if you want me to nix the simple aliasing.

Reply:
great! yea i guess if we're using that aliasing formulation we may as well forward-port it but otherwise kill? not sure what the use-case is